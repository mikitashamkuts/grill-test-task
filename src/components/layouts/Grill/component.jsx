import React from 'react'
import PropTypes from 'prop-types'

import { Paper } from '@material-ui/core'

import GrillInput from '@/components/controls/GrillInput'
import GrillPanel from '@/components/grill/GrillPanel'

function Grill() {
  return (
    <>
      <GrillPanel />
      <GrillInput />
    </>
  )
}

Grill.propTypes = {}

Grill.defaultProps = {}

export default Grill
