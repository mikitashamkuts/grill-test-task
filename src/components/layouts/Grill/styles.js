const styles = theme => ({
  test: {
    padding: theme.spacing(3, 0),
  },
})

export default styles
