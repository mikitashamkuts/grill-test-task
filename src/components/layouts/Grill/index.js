import Grill from './component'

export default Grill