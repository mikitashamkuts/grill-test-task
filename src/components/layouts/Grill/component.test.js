import React from 'react'

import Grill from '.'

describe('Grill layout:', () => {
  test('should match snapshot', () => {
    const component = shallow(<Grill />)
    expect(component).toMatchSnapshot()
  })
})
