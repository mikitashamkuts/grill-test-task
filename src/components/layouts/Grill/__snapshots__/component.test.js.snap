// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grill layout: should match snapshot 1`] = `
<Fragment>
  <WithStyles(GrillPanel) />
  <WithStyles(GrillInput) />
</Fragment>
`;
